.github/workflows/dev_deploy.yml

```diff
@@ -29,6 +29,9 @@ jobs:
       DB_PASS: test
       DB_PORT: "5432"
       DB: test
+      ACCESS_TOKEN_SECRET: 27219441075f8cbcc77b578
```

Comment:
Request @Jkadhuwa to possibly add those values as secrets. While we are on it @Abdulkeza can also ensure to put the required variables into repository secrets and update you on integration regarding the Heroku part.